mne/report.py

```diff
@@ -46,6 +46,33 @@
 # PLOTTING FUNCTIONS
 
 
+def _scene_to_fig(fig):
+    from scipy.misc import imread
+    import matplotlib.pyplot as plt
+    mayavi = None
+    try:
```

Comment:
please share a script so we can test.

also you can share a report online so we can see a demo

thx